llvm/lib/Target/NVPTX/NVPTXISelLowering.cpp

```diff
@@ -94,6 +94,18 @@ static cl::opt<bool> UsePrecSqrtF32(
     cl::desc("NVPTX Specific: 0 use sqrt.approx, 1 use sqrt.rn."),
     cl::init(true));
 
+static cl::opt<bool> UseApproxExp2F32(
```

Comment:
This could use a comment why it's enabled by default, but log2 isn't.

Something along the lines of:
```
Approximate exp2 is enabled by default to match existing CUDA behavior, which always uses `ex2.approx` instruction. This is not the case for `log2`, so it's disabled by default.
```

Reply:
Done.